src/VerifyCustomFactorController.js

```diff
@@ -108,6 +134,19 @@ export default FormController.extend({
     };
   },
 
+  __lastFailedChallengeFactorData__: null,
```

Comment:
Any reason we can't just rely on the presence of appState's `lastFailedChallengeFactorData`?

Reply:
@mauriciocastillosilva-okta @jmelberg-okta 
I was not able to directly access `this.options.appState.get('lastFailedChallengeFactorData')` inside postRender